libs/common/src/assets/i18n/en.json

```diff
@@ -595,6 +594,8 @@
 	"stash.settings.processes.classifier.info": "In order to get accurate label search suggestions, existing content has to be labeled. As more content is labeled, training will improve. You can create new labels and classifications from the ontology page. ",
 	"stash.settings.processes.labeler": " Automatic label training",
 	"stash.settings.processes.labeler.info": "In order to label sentences automatically, existing content has to be labeled. As more content is labeled, training will improve. You can create new labels and classifications from the ontology page. ",
+	"stash.settings.processes.ner": "Named Entities Recognition training",
+	"stash.settings.processes.ner.info": "In order to recognize your custom entities automatically, existing content has to be annotated with your custom entities. As more content is annotated, training will improve. You can create new cutom entities from the Insights page.",
```

Comment:
typo: `You can create new cutom entities from the Insights page.",`
=> `You can create new custom entities from the Insights page.",`

Reply:
fixed